app/main/views/frameworks.py

```diff
@@ -1209,24 +1109,18 @@ def view_contract_variation(framework_slug, variation_slug):
         )
 
         # Send email confirming accepted
-        mandrill_client = DMMandrillClient()
-        email_body = render_template(
-            'emails/{}_variation_{}_agreed.html'.format(framework_slug, variation_slug)
-        )
-        try:
-            mandrill_client.send_email(
-                to_email_addresses=returned_agreement_email_recipients(supplier_framework),
-                email_body=email_body,
-                subject='{}: you have accepted the proposed contract variation'.format(framework['name']),
-                from_email_address=current_app.config['DM_ENQUIRIES_EMAIL_ADDRESS'],
-                from_name=current_app.config['CLARIFICATION_EMAIL_NAME'],
-                tags=['{}-variation-accepted'.format(framework_slug)]
-            )
-        except EmailError as e:
-            current_app.logger.error(
-                "Variation agreed email failed to send: {error}, supplier_id: {supplier_id}",
-                extra={'error': str(e), 'supplier_id': current_user.supplier_id}
-            )
+        notify_client = DMNotifyClient()
+        for address in returned_agreement_email_recipients(supplier_framework):
+            try:
+                notify_client.send_email(
+                    to_email_address=address,
+                    template_name_or_id=notify_client.templates.get(f'{framework_slug}_variation_{variation_slug}'),
```

Comment:
I think leaving the `_agreed` on the end of the key is more descriptive, and keeps in line with the old name of the file.